pkg/lib/cockpit.d.ts

```diff
@@ -247,7 +247,7 @@ declare module 'cockpit' {
     }
 
     interface FileHandle<T> {
-        read(): Promise<T>;
+        read(callback: (data: T | null) => T, tag?: FileTag): Promise<T>;
```

Comment:
cockpit.file().read() does *not* take a callback. And if it would, it'd probably be optional?